modelci/init-mongo.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
```

Comment:
can this replaced by pymongo commands?

Reply:
A more proper way is using a init bash script or a init JS file.
Providing a init file to the container is recommended, see https://stackoverflow.com/a/54064268/13173608

Reply:
Let's use pymongo, it's much better that his solution since the question did indicate the programming language.

Reply:
And their case is for docker container, for us we don't care mongo service is a local service or container service, a python side check will be better.

Reply:
Added. Meanwhile, is it good to keep `init-db.sh`, as this init file can be used in `docker-compose` later?